### PR TITLE
Ignore all but first test in hello-world

### PR DESCRIPTION
**Pull request by jtigger**
Started with @sit's suggestion, but then tacked to use `@Ignore` and defined a copy with filter in the root `build.gradle` that removes these annotations in a maintainers' build.

### CODE REVIEW

**Comment by jtigger** [comment]
This addresses #138 and #140.

File: scripts/increase-test-logging.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (sit) [inline scripts/increase-test-logging.sh]: What are these scripts for? Just for developer use?

Comment (jtigger) [inline scripts/increase-test-logging.sh]: Yes.  They are not at all generalized, but will be.  In the meantime, a starting point to build other scripts?

**Comment by jtigger** [comment]
Thanks for the review @sit!  Yeah, I thought that the added complexity in the maintainer build (which you only have to understand if you need to modify it, at this point) was worth the trade-off of removing the complexity for the exercism programmer.

There are additional benefits.  By turning up test logging, the ignored tests are mentioned, signalling to the programmer there's more to do.

**Comment by matthewmorgan** [comment]
👍 Thanks @jtigger !

**Pull request merged**